### PR TITLE
colorize jobs/variants with zero runs as gray

### PR DESCRIPTION
**Pull request by bparees**
image

### CODE REVIEW

**Comment by bparees** [comment]
/assign @abhinavdahiya 

you've been identified as a potential future sippy maintainer :)

File: pkg/html/releasehtml/top_level_indicators.go [hunk 1]
```diff
@@ -31,10 +31,25 @@ func topLevelIndicators(report, reportPrev sippyprocessingv1.TestReport, release
...
 	tableHTMLTemplate := template.Must(template.New("tableHTML").Parse(tableHTML))
 
-	infraColor := generichtml.OverallInstallUpgradeColors.GetColor(report.TopLevelIndicators.Infrastructure.TestResultAcrossAllJobs.PassPercentage)
-	installColor := generichtml.OverallInstallUpgradeColors.GetColor(report.TopLevelIndicators.Install.TestResultAcrossAllJobs.PassPercentage)
-	upgradeColor := generichtml.OverallInstallUpgradeColors.GetColor(report.TopLevelIndicators.Upgrade.TestResultAcrossAllJobs.PassPercentage)
-	finalColor := generichtml.OverallInstallUpgradeColors.GetColor(report.TopLevelIndicators.FinalOperatorHealth.TestResultAcrossAllJobs.PassPercentage)
+	res := report.TopLevelIndicators.Infrastructure.TestResultAcrossAllJobs
+	passPercent := res.PassPercentage
+	total := res.Successes + res.Failures + res.Flakes
+	infraColor := generichtml.OverallInstallUpgradeColors.GetColor(passPercent, total)
```
Comment (abhinavdahiya) [inline pkg/html/releasehtml/top_level_indicators.go:37]:
```suggestion
	infraColor := generichtml.OverallInstallUpgradeColors.GetColor(
		res.PassPercentage,
		res.Successes+res.Failures+res.Flakes,
	)
```

File: pkg/html/releasehtml/top_level_indicators.go [hunk 2]
```diff
@@ -31,10 +31,25 @@ func topLevelIndicators(report, reportPrev sippyprocessingv1.TestReport, release
...
+
+	res = report.TopLevelIndicators.Install.TestResultAcrossAllJobs
+	passPercent = res.PassPercentage
+	total = res.Successes + res.Failures + res.Flakes
+	installColor := generichtml.OverallInstallUpgradeColors.GetColor(passPercent, total)
```
Comment (abhinavdahiya) [inline pkg/html/releasehtml/top_level_indicators.go:42]:
```suggestion
	installColor := generichtml.OverallInstallUpgradeColors.GetColor((
		res.PassPercentage,
		res.Successes+res.Failures+res.Flakes,
	)
```

File: pkg/html/releasehtml/top_level_indicators.go [hunk 3]
```diff
@@ -31,10 +31,25 @@ func topLevelIndicators(report, reportPrev sippyprocessingv1.TestReport, release
...
+
+	res = report.TopLevelIndicators.Upgrade.TestResultAcrossAllJobs
+	passPercent = res.PassPercentage
+	total = res.Successes + res.Failures + res.Flakes
+	upgradeColor := generichtml.OverallInstallUpgradeColors.GetColor(passPercent, total)
```
Comment (abhinavdahiya) [inline pkg/html/releasehtml/top_level_indicators.go:47]:
```suggestion
	upgradeColor := generichtml.OverallInstallUpgradeColors.GetColor(
		res.PassPercentage,
		res.Successes+res.Failures+res.Flakes,
	)
```

File: pkg/html/releasehtml/top_level_indicators.go [hunk 4]
```diff
@@ -31,10 +31,25 @@ func topLevelIndicators(report, reportPrev sippyprocessingv1.TestReport, release
...
+
+	res = report.TopLevelIndicators.FinalOperatorHealth.TestResultAcrossAllJobs
+	passPercent = res.PassPercentage
+	total = res.Successes + res.Failures + res.Flakes
+	finalColor := generichtml.OverallInstallUpgradeColors.GetColor(passPercent, total)
```
Comment (abhinavdahiya) [inline pkg/html/releasehtml/top_level_indicators.go:52]:
```suggestion
	passPercent = res.PassPercentage
	total = res.Successes + res.Failures + res.Flakes
	finalColor := generichtml.OverallInstallUpgradeColors.GetColor(
		res.PassPercentage,
		res.Successes+res.Failures+res.Flakes,
	)
```

**Pull request merged**